release_dashboard/lib/services/local_conductor.dart

```diff
@@ -18,6 +27,12 @@ import 'conductor.dart';
 ///
 /// This is the production version of the conductor, only intended for releases.
 class LocalConductorService extends ConductorService {
+  LocalConductorService({
+    required this.isDev,
```

Comment:
how about if instead made this a sub-class, `class TestingLocalConductorService extends LocalConductorService`? By having an `if` branch inside `createRelease`, there is essentially two different implementations of this, so I think it would make sense to make them separate classes. This makes it easier to reason about the code, and easier to test.

Reply:
ok sounds good, I will move that. What about the code implementation of local upstreams? Do they make sense?